wulkano/kap
# [WIP] Improve the update notification system

Closes #40 

TODO:

- [ ] The info/refresh icons have, for some reason, different sizes – cc @CodeTheory @skllcrn
- [x] Re-enable auto updates 

How to test it:
1. Go to `app/package.json`
2. Change the `version` to `0.0.0`
3. Run `npm run pack`
4. Open the `.app` that'll be created at `dist/mac`
5. Once the main window is open, open the DevTools (Cmd+Option+I)
6. In a few seconds, you'll see `update available, starting download` in the console
7. Eventually, once the download finishes, the app will send the system notification and the UI will change:

![screen shot 2016-10-13 at 3 07 06 pm](https://cloud.githubusercontent.com/assets/4721750/19360891/cb8a5f70-9156-11e6-97eb-a2c06a83779b.png)

- The ℹ️ icon should open https://github.com/wulkano/kap/releases/latest` in your browser
- The :repeat: icon should restart the app and install the update
